bigquery/data-transfer/api/QuickStart/QuickStart.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Google.Cloud.BigQuery.DataTransfer.V1" Version="1.0.0-beta01" />
```

Comment:
Does this sample require beta01?  Or will it also work with 1.0.0?

Reply:
1.0.0 doesn't exit yet.

> Install the Google.Cloud.BigQuery.DataTransfer.V1 package from NuGet. Add it to your project in the normal way (for example by right-clicking on the project in Visual Studio and choosing "Manage NuGet Packages..."). Please **ensure you enable pre-release packages** (for example, in the Visual Studio NuGet user interface, check the "Include prerelease" box).

https://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.BigQuery.DataTransfer.V1/index.html